libraries/singleton/include/GazeboYarpPlugins/ConfHelpers.hh

```diff
@@ -22,11 +25,6 @@ namespace gazebo
     }
 }
 
-namespace sdf {
-    class Element;
-    typedef boost::shared_ptr<Element> ElementPtr;
-}
```

Comment:
`ElementPtr` is : 
~~~
typedef boost::shared_ptr<Element> ElementPtr;
~~~
in Gazebo < 7 and 
~~~
typedef std::shared_ptr<Element> ElementPtr;
~~~
In Gazebo >= 7 . 
I think it is easier and more readable to just include `#include <sdf/Element.hh>` instead of having a forward declaration with a lot of `#ifdef`.